expected_output1.csv

```diff
@@ -1,36 +1,42 @@
 Kakarott(1) vs Mottaka
 The hero won.
 Kakarott: LVL4
-   HP: 333.3/383.3
-  DMG: 23.3
+   HP: 366.3/383.3
```

Comment:
Még korábban, a levelup feladatnál kellett a HP-t egészre kerekíteni, esetleg ki lehetne íratni kerekítve. Bár nem tudom hirtelen ez változott-e menet közben és már nembaj ha nem egész érték kerül kiírásra... Ezt leszámítva szeintem okés a kód 😉

Reply:
Akkor még úgy is írtam, viszont amikor multiplikatív növekedésről additívra állt át a HP, akkor nem láttam értelmét. Ha nem akarunk tizedesjegyeket, egyszerűen nem használjuk a "health_point_bonus_per_level" attribútumban.

Reply:
Rendben van!